roles/redis_7/defaults/main.yml

```diff
@@ -29,7 +29,7 @@ redis_group: redis
 #
 
 REDIS_REPO: deb https://packages.redis.io/deb {{ ansible_distribution_release }} main
-REDIS_VERSION: 6:7.2.5-1rl1~jammy1
+REDIS_VERSION: 6:7.4.0-1rl1~noble1
```

Comment:
Is it possible to install [Redis 7.2.4](https://github.com/overhangio/tutor/blob/master/tutor/templates/config/defaults.yml#L27) with jammy and noble support?

Reply:
This are the versions available in Jammy:

```
Version: 6:7.0.0-1rl1~jammy1
Version: 6:7.0.1-1rl1~jammy1
Version: 6:7.0.10-1rl1~jammy1
Version: 6:7.0.11-1rl1~jammy1
Version: 6:7.0.12-1rl1~jammy1
Version: 6:7.0.13-1rl1~jammy1
Version: 6:7.0.14-1rl1~jammy1
Version: 6:7.0.15-1rl1~jammy1
Version: 6:7.0.2-1rl1~jammy1
Version: 6:7.0.3-1rl1~jammy1
Version: 6:7.0.4-1rl1~jammy1
Version: 6:7.0.5-1rl1~jammy1
Version: 6:7.0.6-1rl1~jammy1
Version: 6:7.0.7-1rl1~jammy1
Version: 6:7.0.8-1rl1~jammy1
Version: 6:7.0.9-1rl1~jammy1
Version: 6:7.2.0-1rl1~jammy1
Version: 6:7.2.1-1rl1~jammy1
Version: 6:7.2.2-1rl1~jammy1
Version: 6:7.2.3-1rl1~jammy1
Version: 6:7.2.4-1rl1~jammy1
Version: 6:7.2.5-1rl1~jammy1
Version: 6:7.2.6-1rl1~jammy1
Version: 6:7.4.0-1rl1~jammy1
Version: 6:7.4.1-1rl1~jammy1
```

And this are the ones available in Noble:

```
Version: 6:7.2.5-1rl1~noble1
Version: 6:7.2.6-1rl1~noble1
Version: 6:7.4.0-1rl1~noble1
Version: 6:7.4.1-1rl1~noble1
```

Reply:
If redis 7.2.5 was the one installed in the Redwood release instance I think we should probably use that.

Reply:
Agree with that